File: src/main/java/com/atsuishio/superbwarfare/compat/thermoo/ThermooCompatApi.java
package com.atsuishio.superbwarfare.compat.thermoo;

import com.github.thedeathlycow.thermoo.api.temperature.TemperatureAware;
import net.minecraft.server.level.ServerPlayer;

final class ThermooCompatApi {
    static void setTemperature(ServerPlayer player, int temperature) {
        TemperatureAware.get(player).thermoo$setTemperature(temperature);
    }
}

File: src/main/kotlin/com/atsuishio/superbwarfare/compat/thermoo/ThermooCompatApi.kt
package com.atsuishio.superbwarfare.compat.thermoo

import com.github.thedeathlycow.thermoo.api.temperature.TemperatureAware
import net.minecraft.server.level.ServerPlayer

object ThermooCompatApi {
    @JvmStatic
    fun setTemperature(player: ServerPlayer, temperature: Int) {
        TemperatureAware.get(player).`thermoo$setTemperature`(temperature)
    }
}